src/imageops/sample.rs

```diff
@@ -891,6 +958,26 @@ mod tests {
         assert!(img.pixels().eq(resize.pixels()))
     }
 
+    #[test]
+    #[cfg(feature = "png")]
+    fn test_bilinear_sample() {
+        use std::path::Path;
+        let img = crate::open(&Path::new("./examples/fractal.png")).unwrap();
+        assert!(sample_bilinear(&img, 0., 0.).is_some());
```

Comment:
Would probably be good to also have a few test that make sure the interpolation produces the expect values